Library/Formula/libstfl.rb

```diff
@@ -18,6 +18,10 @@ class Libstfl < Formula
   patch :DATA
 
   def install
+    if Formula["ruby"].installed?
```

Comment:
I thought I was missing some "magic" somewhere, heh, but couldn't remember where it was happening. Don't tell anyone but I don't necessarily spend a lot of time with the core code :wink:.

Can we pass `/usr/bin/ruby` to compile to fix this? That seems like the cleanest way to fix all the use cases here, IMO.